src/main/java/com/dev/cinema/dao/impl/UserDaoImpl.java

```diff
@@ -29,8 +29,7 @@ public User add(User user) {
             if (transaction != null) {
                 transaction.rollback();
             }
-            throw new DataProcessingException("Can't add user entity"
-                    + user.getId(), e);
+            throw new DataProcessingException("Can't add user entity", e);
```

Comment:
```suggestion
            throw new DataProcessingException("Can't add user entity: " + user , e);
```
Артьом наступного разу не варто спішити виправляти помилки, подумай як зробити краще

Reply:
сори. плохая идея совмещать с лекцией 🙃